### ISSUE TITLE
Setting en-GB doesn't exclude US English spellings

### ISSUE THREAD

**Issue by TimothyJones** [streetsidesoftware/vscode-spell-checker#2593]
Thanks for this excellent tool!

Using plugin v2.20.0 (VSCode 1.76.1, MacOS 13.2.1 on an Apple M1) and the following configuration:

```
"cSpell.language": "en-GB"
```

### Expected behaviour

In a markdown file, `realise` and `colour`are accepted but `realize` and `color` are not

### Actual behaviour

In a markdown file, all of `realise`, `colour`, `realize` and `color` are accepted. 

If I set the language to:

```
"cSpell.language": "en_US"
```

Then the British English spellings are correctly rejected.

Is there some setting I am missing, or is this a bug?

**Comment by Jason3S** [comment]
@TimothyJones,

A couple of things are going on.

1. The Oxford English dictionary includes a lot of `ize` words.
2. `color` might be included from other dictionaries being used.

To see where a word comes from, the `cspell` command line tool can help. `cspell trace --help` will show all the options.

## Trace `color`

`cspell trace --locale=en-gb --language-id=markdown color`

<img width="824" alt="image" src="https://user-images.githubusercontent.com/3740137/225530718-58cff7ff-ac12-443a-bdaf-41c1cc225ca6.png">

In the case of `color`, markdown is coming from the `html` dictionary. This dictionary is included by default since markdown allows HTML.

If you would like, it is possible to disable the `html` dictionary in markdown files.

The simplest is to add the following to the bottom of the file:

```markdown
<!--- cspell:dictionaries !html --->
```

**Comment by TimothyJones** [comment]
Thanks for the incredibly fast response and debugging information!

That does indeed fix it for my markdown files (although it means that my embedded mermaid diagrams are now complaining about `color`, of course 😂 ).

I reckon this debugging information would be worth including in the FAQ. If you agree, I'd be happy to make a PR.

**Comment by TimothyJones** [comment]
For those playing along at home - omitting the html dictionary didn't fix `realize`, which seems to be coming from `en-gb` (I think?):

![Screenshot 2023-03-16 at 6 18 48 pm](https://user-images.githubusercontent.com/914369/225542937-a982d0dd-9812-422f-bad9-0ffbfd11d8a7.png)

Turns out it's correct - [both are acceptable in British English](https://www.grammarly.com/blog/realise-realize/?gclid=CjwKCAjw_MqgBhAGEiwAnYOAev9U_cjnFeIwU9fQWfZV8GXSll1Kkf_RGi6aOKfnzIcbnkghItkV9hoCDYsQAvD_BwE&gclsrc=aw.ds). TIL

**Comment by Jason3S** [comment]
> I reckon this debugging information would be worth including in the FAQ. If you agree, I'd be happy to make a PR.

Documentation is one of the weak points of this extension, so PRs are most welcome!
